### DISCUSSION TITLE
New flags in Podman v4

### DISCUSSION THREAD

**Discussion by theEndBeta** [containers/ansible-podman-collections#423] [Q&A] [answered]
Are there guidelines on how to add support for flags that are only available as of Podman 4.0? Specifically, I would like to add support for the `--wants` and `--after` flags of [`podman generate systemd`](https://docs.podman.io/en/latest/markdown/podman-generate-systemd.1.html#after-dependency-name). I've made the changes in a fork, but none of the tests are set up for v4 yet, and I don't see them able to be included in default ansible for a while given Ubuntu 22.04 is shipping with v3.4.

**Comment by sshnaidm** [comment, marked as answer]
Done in #431
